x-pack/plugins/ingest_manager/common/services/agent_status.ts

```diff
@@ -41,16 +37,24 @@ export function getAgentStatus(agent: Agent, now: number = Date.now()): AgentSta
   return 'online';
 }
 
+export function buildKueryForEnrolingAgents() {
+  return `not ${AGENT_SAVED_OBJECT_TYPE}.last_checkin:*`;
+}
+
+export function buildKueryForUnenrolingAgents() {
+  return `${AGENT_SAVED_OBJECT_TYPE}.unenrollment_started_at:*`;
+}
```

Comment:
small typos in method names:

```suggestion
export function buildKueryForEnrollingAgents() {
  return `not ${AGENT_SAVED_OBJECT_TYPE}.last_checkin:*`;
}

export function buildKueryForUnenrollingAgents() {
  return `${AGENT_SAVED_OBJECT_TYPE}.unenrollment_started_at:*`;
}
```